src/main/java/frc/robot/commands/Aiming.java

```diff
@@ -27,7 +25,14 @@
   @SuppressWarnings({"PMD.UnusedPrivateField", "PMD.SingularField"})
   private static final Logger LOGGER = Logger.getLogger(DriveCommand.class.getName());
   private Joystick joystick = new Joystick(OI.joystick);
-
+  double tx = SmartDashboard.getNumber("LimelightX", 0);
+  double ty = SmartDashboard.getNumber("LimelightY", 0);
+  double d = 0;
+  
+  double minSteerAdjust = .2;
+  double steeringAdjust = 0.0;
+  double headingCommand = 0;
+  double p = .013;
```

Comment:
Add "final" for d, p and minSteerAdjust to make them constants

Reply:
Also explain in comments where these numbers came from. If they were from experimenting/try and check, just put that.